static/src/javascripts-legacy/projects/common/modules/email/run-checks.js

```diff
@@ -159,7 +159,7 @@ define([
             !clash.userIsInAClashingAbTest(clash.nonEmailClashingTests) &&
             storage.session.isAvailable() &&
             !userHasSeenThisSession() &&
-            !compliantObWidgetIsShown() &&
+            nonCompliantObWidgetIsShown() &&
```

Comment:
Just a small thing...the function call nonCompliantObWidgetIsShown is kind of unnecessary now, you could just use the variable nonCompliantOutbrain in this conditional check. eg. !userHasSeenThisSession() && nonCompliantOutbrain. Then you can remove the function nonCompliantObWidgetIsShown as this is the only place it's called from.